src/framework/constraints.ml

```diff
@@ -612,6 +626,7 @@ struct
   let tf var getl sidel getg sideg edge d =
     begin match edge with
       | Assign (lv,rv) -> tf_assign var edge lv rv
+      | VDecl (v)      -> tf_vdecl var edge v
```

Comment:
Introducing a new kind of edge just for VLAs is a bit heavy. Either solve it differently or fully embrace it and do it for all declarations. Should think about whether this has any (dis)advantages.